packages/strapi-admin/services/permission/sections-builder/handlers.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+const { curry, matchesProperty, pick } = require('lodash/fp');
+
+const utils = {
```

Comment:
why not split them into independent functions or in a handler-helpers file ?

Reply:
Why not indeed, I'll put those function in a dedicated file